go.mod

```diff
@@ -16,3 +16,6 @@ require (
 	go.etcd.io/bbolt v1.3.6
 	google.golang.org/grpc v1.38.0
 )
+
+// Breaking changes were released with the wrong tag (use v0.6.4).
+retract v0.6.5
```

Comment:
woah didnt know this was possible

Reply:
It's a new feature with Go 1.16, but it is really helpful for cases like this. 🙂